--- charybdis-charybdis-3.5.6-orig/configure.ac	2018-08-26 11:50:21.000000000 -0700
+++ charybdis-charybdis-3.5.6/configure.ac	2019-05-10 11:09:49.294634357 -0700
@@ -1058,7 +1058,7 @@
 	# all symbols visible in the dynamic symbol table.
 	hold_ldflags=$LDFLAGS
 	AC_MSG_CHECKING(for the ld -export-dynamic flag)
-	LDFLAGS="${LDFLAGS} -Wl,-export-dynamic -Werror"
+	LDFLAGS="${LDFLAGS} -Wl,-export-dynamic"
 	AC_LINK_IFELSE(AC_LANG_PROGRAM([],[int i;]), found=yes, found=no)
 	LDFLAGS=$hold_ldflags
 
